### ISSUE TITLE
Input normalization

### ISSUE THREAD

**Issue by kurbobo** [zhuduowang/ChangeViT#5]
Hi, thanks for your great work! It is very easy to work with!
As far as I can see from the paper, you use pretrained dino-v2, but there is 0.5 input normalization in your code, that can corrupt the pretrained vit result
Or maybe you retrained dino by yourself?
If you use pretrained dino-v2 by Facebook, then did you tried dino-v2 with registers?
Thank you  in advance

**Comment by zhuduowang** [comment]
Thank you for your interest in our work. Adapting DINOv2 to remote sensing tasks and using 0.5 as the normalization value does not affect the pre-trained parameters, as we fine-tune the entire DINOv2 model. Additionally, we do not use DINOv2 with registers; this remains an area for future exploration.
